plugins/services/src/js/service-configuration/PodGeneralConfigSection.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import ConfigurationMapKeyValueRow from '../components/ConfigurationMapKeyValueRow';
+import ConfigurationMapSection from '../../../../../src/js/components/ConfigurationMapSection';
+import ConfigurationMapHeading from '../../../../../src/js/components/ConfigurationMapHeading';
+import Units from '../../../../../src/js/utils/Units';
+
+function getContainerResourceSummary(resource, appConfig) {
+  let summary = (appConfig.containers || []).reduce((memo, container) => {
+    if (container.resources && container.resources[resource]) {
+      let value = container.resources[resource];
+      memo.value += value;
+      memo.parts.push(
+        `${Units.formatResource(resource, value)} ${container.name}`
+      );
+    }
+    memo.value += (container.resources || {})[resource] || 0;
```

Comment:
please use default assignment. This is getting very hard to follow what is going on